src/commands/purge.ts

```diff
@@ -12,9 +12,10 @@ import {
 } from "discord.js";
 
 const purgeModule: CommandType = {
+  allowGlobal: false,
   data: new SlashCommandBuilder()
     .setName("purge")
-    .setDescription("Purges the last N messages where 1 <= n <= 100")
+    .setDescription("Purges the last N messages where 1 <= n <= 99")
```

Comment:
i purposely left it like this so it can delete the last 100 + the command, your way works too